projects/packages/sync/src/modules/class-woocommerce-products.php

```diff
@@ -292,10 +291,12 @@ public function get_product_by_ids( $ids, $order = '' ) {
 			);
 
 			$post_type = $post->post_type;
+			// ProductType::VARIATION and ProductType::SIMPLE have only existed since WooCommerce 9.7, so
+			// we can't rely on that existing, but using the strings is probably safe enough.
 			if ( 'product_variation' === $post_type ) {
-				$product_type = ProductType::VARIATION;
+				$product_type = 'variation';
 			} elseif ( 'product' === $post_type ) {
-				$product_type = $product_types[ $post->ID ] ?? ProductType::SIMPLE;
+				$product_type = $product_types[ $post->ID ] ?? 'simple';
```

Comment:
This seems fine! I guess we haven't run into any issues here yet, because sync runs on newer stores? or maybe sync has a different plugin requirement?